backend/UserModel.js

```diff
@@ -10,8 +10,16 @@ const userSchema = new Schema({
     type: String,
     require: true,
   },
+  gallery: {
+    type: Array(String),
+    require: false,
+    default: [],
+  },
 });
 
 const User = mongoose.model('User', userSchema);
 
-module.exports = User;
\ No newline at end of file
+// use it for hashing password before saving to database
+userSchema.pre('save', () => {});
```

Comment:
Hash the password before it's saved to the db

Reply:
ok! i will do this in the next PR.